wiki/Article_Styling_Criteria/fr.md

```diff
@@ -0,0 +1,965 @@
+
+---
+
+Les icônes drapeau à côté d'un lien d'utilisateur doit être séparé pour le pseudo de l'utilisateur. Par exemple :
+
+```md
+![][flag_AU] [peppy](https://osu.ppy.sh/users/2)
+```
+
+#### Icônes drapeau 
+
+*Pour une liste d'icone de drapeau, voir : [issue \#328](https://github.com/ppy/osu-wiki/issues/328 "GitHub")*
+
+Toutes les icônes de drapeau utilisent les deux lettres du code du pays (en lettres majuscules) et finissent par `.gif`. Quand vous ajoutez un drapeau intégré, utilisez ce format :
+
+```md
+![](/wiki/shared/flag/xx.gif)
+```
+
+Où `xx` est le code [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 "Wikipedia") du pays pour le nom du drapeau.
+
+Ajouter les noms ou les codes des pays dans le texte alternatif est optionel. Ajouter les noms des pays dans le titre est optionel.
+
+### Tableaux
+
+Les tableaux dans le wiki d'osu! supportent seulement les libellés dans la première ligne. 
+
+Les tableaux ne doivent pas être embellis. Le texte doit être entouré par une barre verticale (`|`) pour chaque cellules du tableau et doit être entouré d'un espace des deux côtés. Les cellules vides doivent utiliser une barre verticale (`|`) suivi de deux espaces puis une autre barre verticale (`|`). 
+
+La ligne de délimitation (la ligne qui sépare la tête de la table de son corps) doivent utiliser trois caractères par colonne (et doivent être espacer des deux côtés), qui ressemblent à un de ceux ci-dessous : 
+
+- `---` (pour un alignement à gauche)
+- `:--` (pour un alignement à gauche)
+- `:-:` (pour un alignement au centre)
+- `--:` (pour un alignement à droite)
+
+Le délimiteur gauche doit être appliqué de manière cohérente dans l’ensemble de l’article.
+
+---
+
+Par exemple :
+
+```md
+| Groupe "Picturesque" rouge | Score | Groupe "Statuesque" Bleu | Moyenne des étoiles des Beatmap |
+| :-- | :-: | --: | :-- |
+| **peppy** | 5 - 2 | pippi | 9.3 stars |
+| Aiko | 1 - 6 | **Alisa** | 4.2 stars |
+| Ryūta | 3 - 4 | **Yuzu** | 5.1 stars |
+| **Taikonator** | 7 - 0 | Tama | 13.37 stars |
+| Maria | No Contest | Mocha |  |
+```
+
+### Blocs de citation
+
+L'usage des blocs de citation est limité à la citation d'une personne. Il ne doit pas être utilisé comme un format de texte.
+
+### Champs thématiques Thematic breaks
+
+Utilisez un champ thématique (aussi connu sous le nom de ligne horizontale) avec partimonie. Quelques exemples des utilisations des champs thématiques (mais ce n'est pas limité qu'à ça) : 
+
+- séparer les images du texte
+- séparer plusieures images qui se suivent
+- changer de sujet dans une section
+
+Quand vous utilisez un champ thématique, soyez sûr qu'il y a une ligne vide au dessus du markup. Autrement, il doit être parsé comme un libellé de niveau 2. Avoir une ligne vide en dessous du markup est préféré mais pas obligatoire. Les champs thématiques doivent avoir seulement trois tiret,comme montré ci-dessous :
+
+```md
+---
+```
+
+## Grammaire
```

Comment:
needs revision as it is meant to be different for every language i think

Reply:
I will do it when we talk about that with other translater and when I have time ^^'